llvm/test/Analysis/BasicAA/matrix-intrinsics.ll

```diff
@@ -0,0 +1,43 @@
+; RUN: opt %s -aa-pipeline=basic-aa -passes=aa-eval -print-all-alias-modref-info -disable-output 2>&1 | FileCheck %s
+
+; BasicAA should prove that loads from sufficiently large static offsets
+; don't overlap with matrix loads with a statically known size.
+
+define <8 x double> @non_overlapping_strided_load(ptr %src) {
```

Comment:
This is just for sload overallping the store, right? The loads both access the same memory, so I think you can remove the second one and just return `%l`?